doxygen warning for \"usersguide\"

**doxygen errors warning**

 doxygen errors for "usersguide" must be fixed before submitting a pull request

defined by CI
[https://travis-ci.com/nasa/cFS/jobs/278425144](url)

**Code snips**
``$ make usersguide > make_usersguide_stdout.txt 2> make_usersguide_stderr.txt
The command "make usersguide > make_usersguide_stdout.txt 2> make_usersguide_stderr.txt" exited with 0.
$ if [[ -s make_usersguide_stderr.txt ]]; then
  echo "You must fix doxygen errors for \"usersguide\" before submitting a pull request"
  echo ""
  cat make_usersguide_stderr.txt
  exit -1``

**matrix:**
 AMD64
 Compiler: gcc C
BUILDTYPE=debug OMIT_DEPRECATED=false

**Additional context**
warning: overriding recipe for target 'usersguide'
warning: ignoring old recipe for target 'usersguide'

**Reporter Info**
Anhelina Yurkova NASA GODDARD / ASRC SES II

--- a/cmake/Makefile.sample
+++ b/cmake/Makefile.sample
@@ -44,6 +44,12 @@
 #  doc -- Build all doxygen source documentation.  The HTML documentation will be
 #      generated under the build tree specified by "O". 
 #
+#  usersguide -- Build all API/Cmd/Tlm doxygen documentation.  The HTML documentation
+#      will be generated under the build tree specified by "O".
+#
+#  osalguide -- Build OSAL API doxygen documentation.  The HTML documentation will
+#      be generated under the build tree specified by "O".
+#
 #  test -- Run all unit tests defined in the build.  Unit tests will typically only
 #      be executable when building with the "SIMULATION=native" option.  Otherwise
 #      it is up to the user to copy the executables to the target and run them.
@@ -70,7 +76,7 @@ endif
 
 # The "LOCALTGTS" defines the top-level targets that are implemented in this makefile
 # Any other target may also be given, in that case it will simply be passed through.
-LOCALTGTS := doc prep all clean install distclean test lcov
+LOCALTGTS := doc usersguide osalguide prep all clean install distclean test lcov
 OTHERTGTS := $(filter-out $(LOCALTGTS),$(MAKECMDGOALS))
 
 # As this makefile does not build any real files, treat everything as a PHONY target

